src/main/java/io/microprofile/config/builtin/converter/BooleanConverter.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright 2016 Microprofile.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package io.microprofile.config.builtin.converter;
+
+import io.microprofile.config.spi.Converter;
+
+public class BooleanConverter implements Converter<Boolean>{
+
+	public Boolean convert(String value) {		
+		return Boolean.valueOf(value);
```

Comment:
This means that only values in the form of "true" and "false" are represented as boolean. Whereas things which are logically more like "on"/"off" and "yes"/"no" aren't?